workflows/diagnostics/tests/prognostic/_regtest_outputs/test_load_diag_data.test_Simulations[simulation0].out

```diff
@@ -1,51 +1,44 @@
 xarray.Dataset {
 dimensions:
-	time = 8 ;
-	x = 48 ;
 	y = 48 ;
+	x = 48 ;
+	time = 8 ;
 	pressure = 35 ;
 	tile = 6 ;
 
 variables:
-	object time(time) ;
-	float32 x(x) ;
 	float32 y(y) ;
+	float32 x(x) ;
+	object time(time) ;
 	float64 pressure(pressure) ;
-	float32 air_temperature(tile, time, y, x, pressure) ;
-		air_temperature:cell_methods = time: point ;
-		air_temperature:long_name = temperature ;
-		air_temperature:units = K ;
-		air_temperature:valid_range = [100.0, 350.0] ;
-	float32 eastward_wind(tile, time, y, x, pressure) ;
-		eastward_wind:cell_methods = time: point ;
-		eastward_wind:long_name = zonal wind ;
-		eastward_wind:units = m/sec ;
-		eastward_wind:valid_range = [-330.0, 330.0] ;
-	float32 northward_wind(tile, time, y, x, pressure) ;
-		northward_wind:cell_methods = time: point ;
-		northward_wind:long_name = meridional wind ;
-		northward_wind:units = m/sec ;
-		northward_wind:valid_range = [-330.0, 330.0] ;
-	float32 pressure_thickness_of_atmospheric_layer(tile, time, y, x, pressure) ;
-		pressure_thickness_of_atmospheric_layer:cell_methods = time: point ;
-		pressure_thickness_of_atmospheric_layer:long_name = pressure thickness ;
-		pressure_thickness_of_atmospheric_layer:units = pa ;
-	float32 specific_humidity(tile, time, y, x, pressure) ;
-		specific_humidity:cell_methods = time: point ;
-		specific_humidity:long_name = specific humidity ;
+	float64 air_temperature(time, tile, y, x, pressure) ;
```

Comment:
I'm guessing these now came from 3D outputs from wrapper instead of Fortran diagnostics. Not a problem.